code/modules/surgery/organ_manipulation.dm

```diff
@@ -80,6 +82,8 @@
 /datum/surgery_step/manipulate_organs/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	manipulated_organ = null
 	if(istype(tool, /obj/item/organ_storage))
+		preop_sound = initial(preop_sound)
+		success_sound = initial(success_sound)
 		if(!tool.contents.len)
 			to_chat(user, "<span class='warning'>There is nothing inside [tool]!</span>")
 			return -1
```

Comment:
I know this isn't your code, but what does -1 even do here. This should probably be a define